kube/services/metadata/metadata-deploy.yaml

```diff
@@ -66,10 +84,18 @@ spec:
         ports:
         - containerPort: 80
         volumeMounts:
-          - name: "config-volume"
+          - name: config-volume-g3auto
+            readOnly: true
+            mountPath: /src/.env
+            subPath: metadata.env
+          - name: config-volume
+            readOnly: true
+            mountPath: /aggregate_config.json
```

Comment:
I'm trying to do one thing at a time, but in conjunction with this PR we should update the `cdis-manifest` entries as needed add an `aggregate_config.json` file per-commons. I'll work on the automated process/init process for importing data automatically on a roll or whatever, but the immediate change would be...

```
kubectl exec $(gen3 pod metadata) -- python /src/src/mds/populate.py --config /aggregate_config.json --hostname esproxy-service --port 9200
```

Reply:
Can we have this in an init-container instead? The init-container can run this before starting the service.

Reply:
If there's no config then init container will exit 0

Reply:
Yeah. I think you're right. I was going to wait to do https://ctds-planx.atlassian.net/browse/HP-287, but I think I may as well do this while I'm updating the rest of it

Reply:
https://github.com/uc-cdis/cloud-automation/pull/1701/commits/18db45d6d64f580181c8f73fd04e24602f6be216